tt_eager/tt_lib/csrc/tt_lib_bindings_tensor.cpp

```diff
@@ -881,6 +881,23 @@ void TensorModule(py::module& m_tensor) {
         +----------+----------------------+-----------+-------------+----------+
     )doc");
 
+    m_tensor.def(
+        "convert_conv_weight_tensor_to_grouped_layout",
+        &convert_conv_weight_tensor_to_grouped_layout,
+        py::arg("conv_weight_tensor").noconvert(),
+        py::arg("num_groups"),
+        py::arg("output_dtype").noconvert() = std::nullopt,
+        R"doc(
+        Converts convolution weights to grouped layout with padded zeros
+        Returns a new tensor with the converted layout.
+
+        +----------+----------------------+-----------+-------------+----------+
+        | Argument | Description          | Data type | Valid range | Required |
+        +==========+======================+===========+=============+==========+
+        | a        | Input tensor         | Tensor    |             | Yes      |
+        +----------+----------------------+-----------+-------------+----------+
+    )doc");
+
```

Comment:
I think we should eventually move these bindings to a lower level module that's not just `ttl.tensor.*`.